packages/cli/commands/validation/themeValidators/themeConfigValidator.js

```diff
@@ -0,0 +1,31 @@
+const { walk } = require('@hubspot/cli-lib/lib/walk');
+const { ERROR_SEVERITY } = require('../validationErrorUtils');
+
+const VALIDATOR_NAME = 'ThemeConfigValidator';
+const THEME_JSON_REGEX = new RegExp(/theme\.json+$/);
+
+//TODO branden do we need this validation? Should we also look at the fields in
+// the theme.json file?
```

Comment:
If we can generate field json schemas that would make it pretty trivial to validate theme.json